nss-day-cohort-48/giffygram-altavista-giffygram
# Jf delete

#### Changes Made

1. Added delete functionally for current logged in users. 
2. Fixed favorites event.
   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout branchname
   ```
2. Open a new Terminal and open the giffygram repo (make sure it's named 'giffygram')
3. Run `./serve-giffygram` to serve the code.
4. Test app functionality.
   >make sure the delete function appears for logged in user.
   > make sure favorite button is working. 
  
5. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
